packages/runtime/datastore/src/test/localChannelStorageService.spec.ts

```diff
@@ -42,7 +43,7 @@ describe("LocalChannelStorageService", () => {
 
         assert.equal(await ss.contains("foo"), true);
         assert.deepStrictEqual(await ss.list(""), ["foo"]);
-        assert.equal(await ss.read("foo"), "bar");
```

Comment:
Note: I'll point out that this code was wrong, as contract of read() was to return base64-encoded strings, but this code clearly expected not encoded string. Nothing to do, just sharing in context of my prior two comments (i.e. looks like IDs were not base64-encoded and the code is good as is)